sysutils/osinfo-db-tools/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                osinfo-db-tools
+version             1.4.0
+platforms           darwin
+categories          sysutils gnome
+license             LGPL-2.1+
+maintainers         nomaintainer
+
+description         Tools for managing the libosinfo database files
+long_description    ${description}
+
+homepage            https://libosinfo.org
+master_sites        https://releases.pagure.org/libosinfo/
+
+checksums           rmd160  212c439f0708ac8c4702f584f263facb70e0b0ce \
+                    sha256  7ee0125c6b99b5b9e10d9bf5951ecc8a170278499a778e6be5c61ef10296f723 \
+                    size    519089
+
+depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    port:gettext \
+                    port:json-glib \
+                    port:libarchive \
+                    port:libxslt
+
+depends_build       port:pkgconfig
```

Comment:
Please move the `depends_build` line before the `depends_lib` line.

Reply:
Done.